src/main/java/com/github/s4u/plugins/PGPVerifyMojo.java

```diff
@@ -312,6 +321,15 @@ private void initCache() throws MojoFailureException {
 
     private boolean verifyPGPSignature(Artifact artifact, File artifactFile, File signatureFile) throws MojoFailureException {
 
+        final Map<Integer, String> weakSignatures = ImmutableMap.<Integer, String>builder()
+                .put(1, "MD5")
+                .put(4, "DOUBLE_SHA")
+                .put(5, "MD2")
+                .put(6, "TIGER_192")
+                .put(7, "HAVAL_5_160")
+                .put(11, "SHA224")
```

Comment:
Ok, thanks. Just noticed it and thought it looked strange but now I know it is a reason behind it.